### ISSUE TITLE
On applying High contrast dark theme, the informational text has only 2.096:1 contrast ratio against its background. :A11y_Template Gallery_Build with Copilot Chat_High Contrast

### ISSUE THREAD

**Issue by Mana-swi** [microsoft/vscode-azurefunctions#5175]
### GitHub Tags
#A11yTCS; #SH_TemplateGallery_Win32_Aug2026; #Azure Tools for VS Code; #DesktopApp; #Win32; #Win11; #A11ySev2; #WCAG1.4.3; #Contrast (Minimum); #a11ydb-custom-contrast; #A11yWCAG;

### Issue Subcategory
Text fails contrast in a high-contrast or dark theme

### User Experience
Users with severe low vision may rely on a product’s custom high-contrast mode specifically to improve readability. Failing contrast in this mode critically impacts usability because this mode may be their last resort.

### Repro Steps
Open VS Code > Install azure functions extension -> Apply VS code High contrast dark theme -> Open command pallet -> Create new project -> Select Build with Copilot Chat

### Actual Result
In the VS Code built-in High Contrast Dark theme, the informational text has a contrast ratio of 2.096:1 against its background, below the required 4.5:1 ratio. The issue does not reproduce with the Windows system high contrast theme.

### Expected Result

The informational text in the custom high-contrast theme must still meet contrast requirements.

### How to Fix
Fix theme color definitions to ensure all text passes contrast checks.

### Environment details
Application Name: Template Gallery (Azure Tools for VS Code)
VS code Version: 1.136.1 (user setup)
Electron: 42.10.0
ElectronBuildId: 15109253
Chromium: 148.0.7778.280
Node.js: 24.18.1
V8: 14.8.178.38-electron.0
OS: Windows_NT x64 10.0.26680

### Note
Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after bug verification.
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Attachment

<img width="853" height="480" alt="Image" src="https://github.com/user-attachments/assets/81d1a75c-5926-489e-8b69-12586dc9d359" />
